shared/network/2.x.md

```diff
@@ -367,6 +367,7 @@ Additionally, you should whitelist the following domains for the relevant ports
 * `*.{{ $names.dashboard_domain }}`
 * `*.docker.com`
 * `*.docker.io`
+* `*.resinio.pool.ntp.org`
```

Comment:
The line above is the main "reviewable" change in this PR (and it has its own commit).
The other commit is the result of fetching external docs.